esm/test/index.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
@sofisl I didn't notice this before, but was the nested `esm` directory and this file intentional?

Reply:
yes!